packages/core/src/auto-args.ts

```diff
@@ -0,0 +1,108 @@
+interface IAuthorArgs {
+  name?: string;
+  email?: string;
+}
+
+export interface IRepoArgs {
+  owner?: string;
+  repo?: string;
+}
+
+export interface ILogArgs {
```

Comment:
Should there be a distinction between `Options` and `Args`?

Reply:
probably not. I'll change this